README.md

```diff
@@ -129,6 +129,7 @@ In addition, if the backends were configured then there will be an environment v
 | ci.pipeline.run.url              | Build URL | String |
 | ci.pipeline.run.user             | Who triggered the build | String |
 | ci.pipeline.run.cause            | List of machine readable build causes like `UserIdCause:anonymous` or `BranchIndexingCause`. Pattern : `${cause.class.simpleName}[:details]` | String |
+| ci.pipeline.run.culprits         | List of users that caused the build. | String |
```

Comment:
Singular or plural?

Reply:
I like plural for string arrays.
Shouldn't this attribute be documented as a `String[]` / String array rather than as a `String`?

Reply:
Uhm, I'm not sure now, since the `cause` uses the same approach, a list of causes, but the attribute name is singular.

Reply:
Good catch. I feel it's `ci.pipeline.run.cause` that should be renamed `ci.pipeline.run.causes` as it's a string array.